results/pkg/server/server.go

```diff
@@ -0,0 +1,272 @@
+package server
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"testing"
+
+	"github.com/golang/protobuf/proto"
+	"github.com/golang/protobuf/ptypes/empty"
+
+	"github.com/google/cel-go/cel"
+	"github.com/google/cel-go/checker/decls"
+	"github.com/google/uuid"
+
+	// create database
+	_ "github.com/mattn/go-sqlite3"
+	pb "github.com/tektoncd/experimental/results/proto/proto"
+	mask "go.chromium.org/luci/common/proto/mask"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+// Server dsadsad
+type Server struct {
+	pb.UnimplementedResultsServer
+	Env *cel.Env
+	Db  *sql.DB
```

Comment:
Once you move the tests back to be in this package, you likely won't need to export these since you already have the `New` function below.

Reply:
done